src/shipit_static_analysis/default.nix

```diff
@@ -123,17 +130,21 @@ let
       export MOZ_AUTOMATION=1
 
       # Use clang mozconfig from gecko-env
-      export MOZCONFIG=${releng_pkgs.gecko-env}/conf/mozconfig
+      export MOZCONFIG=${gecko-env}/conf/mozconfig
 
       # Extras for clang-tidy
       export CPLUS_INCLUDE_PATH=${includes}
       export C_INCLUDE_PATH=${includes}
+
+      # Export linters tools
+      export CODESPELL=${python.packages.codespell}/bin/codespell
```

Comment:
There was also shellcheck, do we need to export it too?

Reply:
This is a bit brittle, as we will need to remember to add them here once they are added on mozilla-central

Reply:
Good idea, that will help